### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier AND"
### and CLHS topic selection, I don't find it
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:and.txt	2017-04-27 21:40:09.000000000 +0100
+++ saved-pages-clus/cl:types:and.txt	2017-04-27 21:40:09.000000000 +0100
@@ -1,4 +1,34 @@
-AND
-Please select which reference to AND you intended
+Type Specifier AND

-MacroType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Combining.
+Compound Type Specifier Syntax
+and typespec*
+Compound Type Specifier Arguments
+typespec - a type specifier.
+Compound Type Specifier Description
+This denotes the set of all objects of the type determined by the intersection of the typespecs.
+* is not permitted as an argument.
+The type specifiers (and) and t are equivalent. The symbol and is not valid as a type specifier, and, specifically, it is not an abbreviation for (and).
+\
\ No newline at end of file
